packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -83,6 +83,22 @@ export const toVector3D = (v: Vector3) => {
   };
 };
 
+export interface Vector2D {
+  x: number
+  y: number
```

Comment:
Should this be u and v instead?

Reply:
I changed that now, but wouldn't it be maybe better for other use cases in the future if it stays x and y? So it can be reused without creating confusion?

Reply:
Maybe, but I think we'll cross that bridge when we come to it. I don't see a lot of need to return `{x, y}`, so hopefully it's a non-issue.